db/models.py

```diff
@@ -22,6 +24,11 @@ class Movie(models.Model):
     actors = models.ManyToManyField(to=Actor, related_name="movies")
     genres = models.ManyToManyField(to=Genre, related_name="movies")
 
+    class Meta:
+        indexes = [
+            models.Index(fields=["title"], name="muvie_title_idx")
```

Comment:
There is a typo in the index name 'muvie_title_idx'. It should be 'movie_title_idx' to match the correct spelling of 'movie'.